domains/grid/utils/gridParser.ts

```diff
@@ -1,7 +1,7 @@
-export const parsePlatformInput = (
+export const parsePlatformInput = async (
   platform: GridWidgetType,
   input: string
-): LayoutItemExtended | never => {
+): Promise<LayoutItemExtended> => {
```

Comment:
I think you miss await in the function. Also why `never` type was removed?

Reply:
good catch